Assets/Scripts/Craft/Requirement.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Bug.Craft
+{
+    [Serializable]
+    public class Requirement
```

Comment:
Still need this after linking to the inventory system?

Reply:
Nope, I'll delete it